keras_cv/models/object_detection/faster_rcnn/faster_rcnn.py

```diff
@@ -231,26 +61,17 @@ class FasterRCNN(keras.Model):
     References:
         - [FasterRCNN](https://arxiv.org/pdf/1506.01497.pdf)
 
-    Usage:
-    ```python
-    retinanet = keras_cv.models.FasterRCNN(
-        num_classes=20,
-        bounding_box_format="xywh",
-        backbone=None,
-    )
-    ```
-
     Args:
+        backbone: `keras.Model`. Must implement the
+            `pyramid_level_inputs` property with keys "P2", "P3", "P4", and "P5"
+            and layer names as values. If `None`, defaults to
```

Comment:
The `None` part is no longer relevant  -- we should (and it looks like the code does) require a backbone argument